src/main/java/org/openrewrite/java/spring/boot3/AddMappingPathsWithTrailingSlash.java

```diff
@@ -0,0 +1,145 @@
+                        }
+
+                    } else {
+                        // search for value
+                        List<Expression> args = anno.getArguments();
+                        for (int i = 0 ; i < args.size(); i++) {
+                            Expression exp = args.get(i);
+                            if (exp instanceof J.Assignment) {
+                                J.Assignment assignment = (J.Assignment) exp;
+                                if (assignment.getVariable() instanceof J.Identifier &&
+                                    ((J.Identifier) assignment.getVariable()).getSimpleName().equals("value") &&
+                                    isStringLiteral(assignment.getAssignment()) ) {
+
+                                    J.Literal str = (J.Literal) assignment.getAssignment();
+                                    if (!matchTrailingSlash(str.getValue().toString())) {
+                                        args.set(i, buildAssignment(str));
+                                        return annotation.withArguments(args);
+                                    }
+                                }
+                            }
+                        }
+
+                    }
+                }
+
+                return anno;
+            }
+        };
+    }
+
+    private boolean matchTrailingSlash(String str) {
+        return str.endsWith("/") || str.endsWith("*");
+    }
+
+    private J.Assignment buildAssignment(J.Literal path) {
+        String oriPath = path.getValue().toString();
+        String pathWithTrailingSlash = oriPath + '/';
+
+        J.Assignment assignmentTemplate = getAssignmentTemplate().withPrefix(Space.EMPTY);
+        J.NewArray twoPaths = getTwoStringsArrayTemplate();
+        List<Expression> exps = twoPaths.getInitializer();
+        exps.set(0, path.withPrefix(EMPTY));
+        exps.set(1, path.withValue(pathWithTrailingSlash)
+            .withValueSource("\"" + pathWithTrailingSlash + "\"")
+            .withPrefix(Space.build(" ", emptyList())));
+        twoPaths = twoPaths.withInitializer(exps);
+        return assignmentTemplate.withAssignment(twoPaths);
+    }
+
+    private static boolean isHttpVerbMappingAnnotation(String fqn) {
+        return GET_ANNOTATION_TYPE.equals(fqn) ||
+               REQUEST_ANNOTATION_TYPE.equals(fqn) ||
+               POST_ANNOTATION_TYPE.equals(fqn) ||
+               PUT_ANNOTATION_TYPE.equals(fqn) ||
+               PATCH_ANNOTATION_TYPE.equals(fqn) ||
+               DELETE_ANNOTATION_TYPE.equals(fqn);
+    }
+
+    private static J.NewArray getTwoStringsArrayTemplate() {
+        if (twoStringsArrayTemplate == null) {
+            twoStringsArrayTemplate = PartProvider.buildPart("class Test {\n" +
+                                                                "    String[] value = { \"a\", \"b\"};\n" +
+                                                                "}",
```

Comment:
For legibility I would put the string literal on the next line so the code snippet isn't indented so far to the right